Detectors/TOF/prototyping/drawTOFgeometry.C

```diff
@@ -0,0 +1,105 @@
+#if (!defined(__CINT__) && !defined(__CLING__)) || defined(__MAKECINT__)
```

Comment:
This is missing the copyright statement. Can you please add it? I think we are not yet checking for copyright in macros, but we probably should.